docker-compose.yml

```diff
@@ -1,28 +1,63 @@
 version: '2'
 services:
   bluetooth-audio:
-    network_mode: host
-    restart: always
     build: ./bluetooth-audio
+    restart: always
+    network_mode: host
     privileged: true
     labels:
       io.balena.features.dbus: 1
     volumes:
       - bluetoothcache:/var/cache/bluetooth
+      - snapcast:/var/cache/snapcast
   airplay:
     build: ./airplay
     restart: always
     network_mode: host
     privileged: true
     labels:
       io.balena.features.dbus: 1
+    volumes:
+      - snapcast:/var/cache/snapcast
   spotify:
     build: ./spotify
     restart: always
     network_mode: host
     privileged: true
     volumes:
       - spotifycache:/var/cache/raspotify
+      - snapcast:/var/cache/snapcast
+  fleet-supervisor:
+    build: ./fleet-supervisor
+    restart: always
+    network_mode: host
```

Comment:
We should consider reviewing the containers to make sure they all need `network_mode: host` and privileged where specified - but that can be a separate thing, no need to block

Reply:
I reviewed it already before submitting the PR. I can't remember on top of my head the reasons for all of them, but for example for `fleet-supervisor` I could not get `cote.js` library to work without it. That also forces `snapcast-client` to be in host mode because it can otherwise hit the server on the exposed port. 
I'll add this to re test when I do the `bluetooth-audio` split.